SparkPost/support-docs
# WE-286: 404 page

### What Changed

The 404 page has been added.

### How To Test or Verify
- either use the preview link on netlify or go to `localhost:3000/not-a-page`
- verify the 404 page matches design - https://www.figma.com/file/Hsjkp15919WQeDkpxWBa5G/Support-Docs-404-Page?node-id=0%3A1

### PR Checklist
- [x] Pull request approval from #proj-fix-support-docs
